Widgets/widget.css

```diff
@@ -0,0 +1,61 @@
+
+body {
+   background-color:black;
+   font-family: 'Cantarell';
+   color:white;
+   border: solid #23c1e8 4px;
+   background-color: #194d6c;
+}
+
+/* Headers */ 
+
+h1 {
+   font-size: 2em;
+   font-weight:  bold;
+   margin: 0.2em;
```

Comment:
it might be better to leave margins to the discretion of the individual widgets... ie not put them on a commonly-used thing like an h2. but i'm not 100% sure....